### PR TITLE
Metrics webapp

### PR DESCRIPTION
**Pull request by errea**
# Description

This React capstone project is about building a mobile web application to check a list of metrics (numeric values) that you will create making use of React and Redux. I selected an [API](https://covid19tracking.narrativa.com/index_en.html) that provides numeric data about a topic that you like and then build the web app around it. This PR is to check for the minimum request to make this project an MVP.

## Project

Project Descriptions and Requirments 

- [x] Use React documentation.
- [x] Use React components.
- [x] Use React props.
- [x] Use React Router.
- [x] Connect React and Redux.
- [x] Handle events in a React app.
- [x] Write integration tests with a React testing library.
- [x] Use styles in a React app.
- [x] Use React life cycle methods.
- [x] Apply React best practices and language style guides in code.
- [x] Use store, actions, and reducers in React.
- [x] JavaScript/ES6 best practices 
- [x] HTML/ CSS best practices 

## Project Demo

Screen Shot 2021-09-09 at 12 19 54 PM

# How Has This Been Tested?

Please describe the tests that you ran to verify your changes. Provide instructions so we can reproduce. Please also list any relevant details for your test configuration

- [x] Jest

**Test Configuration**:
* Firmware version:
* Hardware:
* Toolchain:
* SDK:

# Checklist:

- [x] My code follows the style guidelines of this project
- [x] I have performed a self-review of my own code
- [x] I have commented on my code, particularly in hard-to-understand areas
- [x] I have made corresponding changes to the documentation
- [x] I have added tests that prove my fix is effective or that my feature works
- [x] New and existing unit tests pass locally with my changes

### CODE REVIEW

File: src/components/Header.js [hunk 1]
```diff
@@ -0,0 +1,22 @@
...
+  return (
+    <header className="header">
+      {location.pathname === '/details'
+        ? <button type="button" aria-label="Go back" className="header-back-icon" onClick={() => history.goBack()}><ChevronLeftIcon /></button>
+        : <div />}
+      <div>
+        <h1 className="header-title">COVID Data Metircs App</h1>
+      </div>
+      <div />
+    </header>
```
Comment (Dipeshtwis) [inline src/components/Header.js:18]: - [x] The arrow is overlapping with the text from 320px to 370px. Please fix that.

Screenshot from 2021-09-09 23-21-22

**Review by Dipeshtwis — CHANGES_REQUESTED** [review]
Hi @errea, 

:+1: This is good to make it this far, your project is really nice, congratulations on the good work you have done so far.
:+1: Deployed the application using Netflix and working as expected.
:+1: The App used redux to store the data. Well Done.

Good job so far!
There are some issues that you still need to work on to go to the next project but you are almost there!


 ## Required Changes ♻️

**_Check the comments under the review_**

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

- N.A.

Cheers and Happy coding!👏👏👏
Feel free to leave any questions or comments in the PR thread if something is not 100% clear.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

**_Dipesh Kumar_**


------

_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project(check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: src/components/Header.js (hunk 1 above)
Comment (errea) [reply src/components/Header.js:18]: Fixed

**Review by arslanbisharat — APPROVED** [review]
# Approved 👏 

Hi there, 

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 


Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of code reviews per project which you can see from your dashboard. If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)_

**Pull request merged**